gradle-conjure-api/src/main/java/com/palantir/gradle/conjure/api/ConjureExtension.java

```diff
@@ -45,6 +46,11 @@ public final void python(@DelegatesTo(GeneratorOptions.class) Closure<GeneratorO
         closure.call();
     }
 
+    public final void parser(@DelegatesTo(GeneratorOptions.class) Closure<GeneratorOptions> closure) {
```

Comment:
You should `Closure` arguments as raw types for the gradle integration in intellij to work properly (without making a big warning box). Also, for `Closure<V>`, `V` is the return type so it shouldn't really be the delegate type.

Reply:
Probably a nice thing to make an errorprone rule of actually

Reply:
Huh, interesting! I copied from the other methods on this object, I'll update all of 'em.

Reply:
I only just discovered this recently too!